src/runtime_src/core/pcie/driver/linux/xocl/mgmtpf/mgmt-core.c

```diff
@@ -493,6 +493,8 @@ static int health_check_cb(void *data)
 
 	(void) xocl_clock_status(lro, &latched);
 
+	xocl_xmc_clock_status(lro, &latched);
+
```

Comment:
Can you please add a comment explaining why do we need to add a separate logic for U.2. example: U.2 doesn't use UCS etc